core/src/batch/filter_map.rs

```diff
@@ -1,23 +1,32 @@
 use super::{Batch, Disposition};
 use crate::packets::Packet;
-use crate::Result;
+use crate::{Mbuf, Result};
+
+/// The outcome of the filter map.
+pub enum Outcome<T> {
```

Comment:
part of me wanted something like EitherOutcome (just to reference the concept of Either here), but no biggie either way.

Reply:
hmmm, I can name it just `Either` if you really prefer. `EitherOutcome` feels a bit too much to write :) and making it a slightly clunky to use.